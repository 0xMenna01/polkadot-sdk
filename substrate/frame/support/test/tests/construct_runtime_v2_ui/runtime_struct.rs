#[frame_support::construct_runtime_v2]
mod runtime {
    #[frame::runtime]
    pub enum Runtime {}
}

fn main() {}
