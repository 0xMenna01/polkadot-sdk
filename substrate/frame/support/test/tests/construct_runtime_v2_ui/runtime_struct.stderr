error: Invalid frame::runtime, expected struct definition
 --> tests/construct_runtime_v2_ui/runtime_struct.rs:4:5
  |
4 |     pub enum Runtime {}
  |     ^^^
